tests/cli/test_init_sqlite.py

```diff
@@ -0,0 +1,375 @@
+import os
+import re
+import shutil
+
+import pytest
+from click.testing import CliRunner
+from sqlalchemy import create_engine
+
+from great_expectations import DataContext
+from great_expectations.cli import cli
+from great_expectations.data_context.util import file_relative_path
+from great_expectations.util import gen_directory_tree_str
+from tests.cli.test_cli import yaml
+
+@pytest.fixture
+def titanic_sqlite_db_file():
+    from sqlalchemy import create_engine
+    titanic_db_path = file_relative_path(__file__, "../test_sets/titanic.db")
+    engine = create_engine('sqlite:///{}'.format(titanic_db_path))
+    assert engine.execute("select count(*) from titanic").fetchall()[0] == (1313,)
+    return titanic_db_path
```

Comment:
I like this fixture...I hadn't thought of it, but another meta-note @eugmandel : this approach would be useful for the Quality Improvement effort -- saving db dumps instead of loading/unloading because we'd have more control on how the DB was designed without having to keep infrastructure up.